core/src/main/java/lucee/runtime/functions/closure/Reduce.java

```diff
@@ -55,7 +55,11 @@ public static Object call(PageContext pc, Object obj, UDF udf, Object initalValu
 		return _call(pc, obj, udf, initalValue, TYPE_UNDEFINED);
 	}
 
-	public static Object _call(PageContext pc, Object obj, UDF udf, Object initalValue, short type) throws PageException {
+	public static Object call(PageContext pc, Object obj, UDF udf, Object initalValue, short type) throws PageException {
+		return _call(pc, obj, udf, initalValue, type);
```

Comment:
why that change?

Reply:
just to be consistent, everything starting with _ is private